packages/block-editor/src/components/inserter/menu.js

```diff
@@ -356,7 +358,33 @@ export class InserterMenu extends Component {
 							</PanelBody>
 						) }
 						{ isEmpty( suggestedItems ) && isEmpty( reusableItems ) && isEmpty( itemsPerCategory ) && (
-							<p className="editor-inserter__no-results block-editor-inserter__no-results">{ __( 'No blocks found.' ) }</p>
+							<p className="editor-inserter__no-results block-editor-inserter__no-results">
+								{ numberOfHiddenBlocks ? __( 'No active blocks found.' ) : __( 'No blocks found.' ) }
+								{ !! numberOfHiddenBlocks && (
+									<>
+										<br />
+										{
+											sprintf(
+												_n(
+													'%1$d block is disabled.',
+													'%1$d blocks are disabled.',
+													numberOfHiddenBlocks
+												),
+												numberOfHiddenBlocks
+											)
+										}
+										<br />
+										<Button
+											isButton
+											isDefault
+											className="editor-inserter__block-manager-button block-editor-inserter__block-manager-button"
```

Comment:
There should be only one class name here, the obe starting with „block-editor”. The other one exists only for backward compatibility in other places.

Reply:
Fixed, thanks! I knew one was older but wasn't sure if I was supposed to still include it or not.